build/NuGetPackages/Microsoft.Build.Conversion.Core.nuspec

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
```

Comment:
Yeah I think I'm going to just delete the one's under src/nuget.  These will replace them anyway and the only reason their under src/nuget is to use BuildTools which we can't really use anymore with this change. 

---
In reply to: [78386596](https://github.com/Microsoft/msbuild/pull/1013#discussion_r78386596) [](ancestors = 78386596)